backend/src/Altinn.Authorizationadmin/Altinn.Authorizationadmin/Migration/v0.07-authorization-admin/02-setup-procedures.sql

```diff
@@ -1,22 +1,22 @@
 --Procedure: delegation.InsertResourceDelegation
-CREATE OR REPLACE FUNCTION delegation.InsertResourceDelegation(
-  _delegationChangeType delegation.delegationChangeType,
-  _altinnappid character varying,
-  _offeredbypartyid integer,
-  _coveredbyuserid integer,
-  _coveredbypartyid integer,
-  _performedbyuserid integer,
-  _blobstoragepolicypath character varying,
-  _blobstorageversionid character varying,
-  _resourceid character varying,
-  _resourcetype character varying
-)
-RETURNS SETOF delegation.delegationchanges 
-LANGUAGE 'sql'
-VOLATILE
-ROWS 1
+CREATE OR REPLACE FUNCTION delegation.insert_delegationchange(
+	_delegationchangetype delegation.delegationchangetype,
+	_altinnappid character varying,
+	_offeredbypartyid integer,
+	_coveredbyuserid integer,
+	_coveredbypartyid integer,
+	_performedbyuserid integer,
+	_blobstoragepolicypath character varying,
+	_blobstorageversionid character varying,
+	_resourceid character varying,
+	_resourcetype character varying)
```

Comment:
_resourcetype is defined as a Type in the resourceregistry schema. We may not be able to reference it directly but should in stead make our own Type in the delegation schema?